### DISCUSSION TITLE
AH02429: Response header name '[2022-02-28T05' contains invalid characters | firefly 5.6.16

### DISCUSSION THREAD

**Discussion by GaneshKandu** [firefly-iii/firefly-iii#5835] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hi,

I am getting **Internal Server Error** when accessing firefly 5.6.15 after install.
Exactly after setting user credentials

Apache error log
```AH02429: Response header name '28 02 2022' contains invalid characters, aborting request``` 

probably duplicate of #4074 .

### Expected behaviour

I do not Expect **Internal Server Error**

### Debug information

Blank page on /debug

Apache Log

```AH02429: Response header name '[2022-02-28T05' contains invalid characters, aborting request```

### Steps to reproduce

1. composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist firefly5616 5.6.16
2. sudo chown -R www-data:www-data firefly-iii
3.  sudo chmod -R 775 firefly-iii/storage
4.  Fill up details in .env
5.  php artisan key:generate
6.  php artisan migrate:refresh --seed
7. visit [http://localhost/firefly-iii/](http://localhost/firefly-iii/public)
8. set admin credentials
9. **Internal Server Error**

### Additional info

I am using php 8.0.14
Apache 2.4.52

#### I tried to reproduce similar issue.

```php
<?php

header('Content-Type: text/plain; charset=UTF-8');
header('foo: bar');
header('a: b');
header('['.date(DATE_ATOM,time()).']'.': b');

print_r(headers_list());
```
this code causing exactly similar issue i am getting in firefly.

```header('['.date(DATE_ATOM,time()).']'.': b');```

i think you initializing some variable with date value enclosed between opening and closing squire brackets for example ```[2022-02-28T05:49:11.847659-05:00]```
```[2022-02-28T05:49:11.847659-05:00] local.INFO: AUDIT: Show login form (1.1).```

and you passing it in ```header``` function also as key somewhere in your code for example.
```php
header('['.date(DATE_ATOM,time()).']'.': b');
```
that causing **Internal Server Error**

**Comment by JC5** [comment]
Thanks for the research. The root cause is not that I'm sending headers. In some configurations the audit logs ends up in the php output buffer which breaks Apache. 

In the .env file you can find instructions to set the audit log level to "emergency". That will solve it.

**Comment by GaneshKandu** [comment]
setting the audit log level to "emergency" works,
but what does it do? does it change any functionality ?

**Reply by JC5** [reply]
Nothing important
